### PR TITLE
[qt] Satoshi unit

### PR DESCRIPTION
**Pull request by GreatSock**
This adds satoshi as an additional amount unit for the GUI.

### CODE REVIEW

**Comment by kristapsk** [comment]
utACK c722f00a7e3d638253ef7ba59fcef123fcfd94b0

**Comment by ken2812221** [comment]
Concept ACK

File: src/qt/bitcoinunits.cpp [hunk 1]
```diff
@@ -52,7 +55,8 @@ QString BitcoinUnits::shortName(int unit)
     switch(unit)
     {
     case uBTC: return QString::fromUtf8("bits");
```
[the remark this answers is not in the document]
Comment (GreatSock) [reply src/qt/bitcoinunits.cpp]: Seems so. Should I make a commit to get rid of it?

[the remark this answers is not in the document]
Comment (laanwj) [reply src/qt/bitcoinunits.cpp]: it's unnecessary but also doesn't really matter, there's no more overhead than `QString(...)` and anyway, it's not a performance critical function.

**Comment by laanwj** [comment]
utACK c722f00a7e3d638253ef7ba59fcef123fcfd94b0

**Comment by MarcoFalke** [comment]
utACK c722f00

**Review by kallewoof — APPROVED** [review]
utACK c722f00a7e3d638253ef7ba59fcef123fcfd94b0

**Comment by jonasschnelli** [comment]
Tested ACK c722f00a7e3d638253ef7ba59fcef123fcfd94b0

**Pull request merged**